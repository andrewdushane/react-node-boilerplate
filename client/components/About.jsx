import React from 'react';

const About = () => (
  <div>
    About this App
  </div>
);

export default About;
