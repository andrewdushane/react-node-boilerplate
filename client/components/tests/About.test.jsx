import React from 'react';
import renderer from 'react-test-renderer';
import About from '../About';

test('About renders as expected', () => {
  const component = renderer.create(
    <About />
  );
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
