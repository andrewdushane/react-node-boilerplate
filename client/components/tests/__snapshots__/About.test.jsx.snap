exports[`test About renders as expected 1`] = `
<div>
  About this App
</div>
`;
